### ISSUE TITLE
[Feature] Selection based regex generator using Web UI 

### ISSUE THREAD

**Issue by bauthard** [projectdiscovery/nuclei-templates#467]
**Is your feature request related to a problem? Please describe.**

Sometimes it's not easy to write the right regex to match your need or not everyone is good with regex. 

**Describe the solution you'd like**

To make this process simple, we can have a Webform, where one can paste the response data, and the make the selection on the data they wish to extract, and based on selection, a regex will be generated that can be used with nuclei.

**Comment by 6t2** [comment]
Hello, any updates on this? :)

**Comment by ehrishirajsharma** [comment]
Yes, we are tracking and working on this, should be released by the next month.
